tests/integration/orcid/test_orcid_domain_models.py

```diff
@@ -138,6 +138,13 @@ def test_push_updated_work_happy_flow(self):
         assert result_putcode == int(self.putcode)
         assert not self.cache.has_work_content_changed(self.inspire_record)
 
+    def test_push_updated_work_no_cache(self):
```

Comment:
@drjova could you record the cassette for me?

Reply:
:( I'll